docker-compose.yml

```diff
@@ -1,48 +1,52 @@
 version: '2'
-services: 
-    xnat-web:
-       build: ./tomcat
-       ports:
-        - "8081:8080"
-       volumes:
-         - ./webapps:/opt/tomcat/webapps
-       expose:
-        - "8080"
-       links:
-        - xnat-db    
-    xnat-db:
-       build: ./postgres
-       expose:
-         - "5432"
-       volumes:
-         - ./postgres-data:/var/lib/postgresql/data
-    xnat-nginx:
-       build: ./nginx
-       ports:
-        - "80:80"
-       expose: 
-        - "80"
-       links:
-         - xnat-web
+services:
```

Comment:
A lot of the "changes" in this file are just whitespace. Can you revert these changes?